src/EPR.Common/NuGet.Config

```diff
@@ -3,6 +3,5 @@
 	<packageSources>
```

Comment:
seems there's no build on this repo - holding off merging till that's sorted

(comment on arbitrary code just so the comment can be "resolved")